lib/cordial/products.rb

```diff
@@ -5,6 +5,13 @@ class Products
     include ::HTTParty
     extend Client
 
+    # Find all products
+    # @example Usage
+    #  Cordial::Products.all
+    def self.all
```

Comment:
What do you think about making this `index` to match Orders. As this gem matures, we'll have more examples and slowly hone in on what we want the patterns to be.

Reply:
Updated 😄